pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -92,6 +99,7 @@ def emit_column_headers(self) -> None:
         self.emit_text("     - PEP")
         self.emit_text("     - Title")
         self.emit_text("     - Authors")
+        self.emit_text("     - Python")
```

Comment:
"Python version" would be a clearer title but it is much longer than the actual contents of the entries however. :/

Reply:
Yeah... I put "Version" at first but "Python" is a bit shorter.

We could put something like "v." or "v", or even leave it blank, as it should be obvious what it is?

Reply:
I'd vote for leaving it blank

Reply:
Updated.